tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,13 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
This works, but it requires that the implementation of `swap_best_by_category` doesn't make changes to the order of the items, and also that the new items are added at the end of the lists.  It's longer to write out, but I would suggest using a series of `assert item in [person].inventory` statements.